docs/antora/modules/ROOT/pages/configuration/source.adoc

```diff
@@ -12,137 +12,235 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_pm2_ecosystem_file: https://pm2.keymetrics.io/docs/usage/application-declaration/
 
 :url_management_pm2: management/source.adoc
-:url_references_config: reference/configuration.adoc
+:url_references_config: configuration/index.adoc
 
-== Default configuration
+// The following Default section commented out below has now been updated with the latest values for later use.
```

Comment:
Why is the default configuration section removed/ commented out completely?

Reply:
Ah, this is because the default configuration is covered in the previous page (the index page). 
In the TIP on this page it refers/links to the default config in the index page.

Reply:
I see, thanks!